src/bin/vip.js

```diff
@@ -36,7 +36,7 @@ const runCmd = async function () {
 		.command( 'logs', 'Get logs from your VIP applications' )
 		.command( 'search-replace', 'Perform search and replace tasks on files' )
 		.command( 'slowlogs', 'Get slowlogs from your VIP applications' )
-		.command( 'db', 'Run operations on your VIP application database' )
+		.command( 'db', 'Connect to an environment database.' )
```

Comment:
Since there are currently no other commands under `vip db,` should we provide clearer visibility that the `db` command is the access point to `vip db phpmyadmin` by referencing phpMyAdmin in the top-level command description?

> Connect to an environment database using phpMyAdmin.